apps/base/utils.py

```diff
@@ -168,6 +177,25 @@ def get_boto3_client(resource, aws_keys):
         logger.exception(e)
 
 
+def get_presigned_url_for_file_upload(file_key):
+    """
+    Function to get the presigned url to upload a file to s3 with name file_name and key as file_key.
+    """
+    try:
+        aws_keys = get_aws_secret_keys()
+
+        s3 = get_boto3_client("s3", aws_keys)
+        response = s3.generate_presigned_post(
+            Bucket=settings.AWS_STORAGE_BUCKET_NAME,
```

Comment:
why are you using the bucket explicitly from settings? shouldn't this also go in `get_aws_secret_keys`